docs/CONTRIBUTING.md

```diff
@@ -288,7 +285,7 @@ Then you can hit load button and choose `thread_profile.json` file.
 [di]: https://discord.gg/amethyst
 [bs]: https://www.kth.se/social/upload/5289cb3ff276542440dd668c/bitsquid-behind-the-scenes.pdf
 [fr]: http://twvideo01.ubm-us.net/o1/vault/gdc2012/slides/Programming%20Track/Persson_Tobias_Flexible_Rendering.pdf.pdf
-[ma]: http://www.amd.com/Documents/Mantle-Programming-Guide-and-API-Reference.pdf
+[ma]: https://www.yumpu.com/en/document/view/43374261/mantle-programming-guide-and-api-reference
 [mo]: http://shaneenishry.com/blog/2014/12/27/misconceptions-of-component-based-entity-systems/
```

Comment:
This is the blog post link that is broken and I can't find a replacement for.

Reply:
here you go :v:: https://shanee.io/blog/2014/12/27/misconceptions-of-component-based-entity-systems/